lab-sarah/gulpfile.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+
+gulp.task('hello', function(){
```

Comment:
If you're copying a generic gulpfile for all of your projects, I would remove this task as it's essentially useless ;)